services/wordpress/wordpress-base.js

```diff
@@ -4,25 +4,48 @@ const Joi = require('@hapi/joi')
 const { nonNegativeInteger } = require('../validators')
 const { BaseJsonService, NotFound } = require('..')
 
-const foundSchema = Joi.object()
+const stringOrFalse = Joi.alternatives(Joi.string(), Joi.bool())
+
+const themeSchema = Joi.object()
   .keys({
-    version: Joi.string(),
-    rating: nonNegativeInteger,
-    num_ratings: nonNegativeInteger,
-    downloaded: nonNegativeInteger,
-    active_installs: nonNegativeInteger,
-    requires: Joi.string(), // Plugin Only
-    tested: Joi.string(), // Plugin Only
+    version: Joi.string().required(),
+    rating: nonNegativeInteger.required(),
+    num_ratings: nonNegativeInteger.required(),
+    downloaded: nonNegativeInteger.required(),
+    active_installs: nonNegativeInteger.required(),
```

Comment:
the `nonNegativeInteger` helper already includes `required()` so there's no need to add a duplicative one here
```suggestion
    rating: nonNegativeInteger,
    num_ratings: nonNegativeInteger,
    downloaded: nonNegativeInteger,
    active_installs: nonNegativeInteger,
```